### PR TITLE
Product page UI

### PR DESCRIPTION
**Pull request by abhijitdotsharma**
Product Page UI
- Cards with data from mock server
- Filter component
- navigation component

#### p.s - Filter functionality to be added on next PR
temp-github-noustoreReact-pr

### CODE REVIEW

**Review by neetukumari4858 — APPROVED** [review]
overall looks good 
code is readable 
UI looks good to see

File: src/components/products/ProductCard.jsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
+//I will take a product detail prop from Products.jsx and render is beautifully
+import { useState } from "react";
+
+const ProductCard = ({ product }) => {
+
```
Comment (sushil-mod) [inline src/components/products/ProductCard.jsx:5]: you destructure product to price ,photo etc

File: src/context/product-context.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        //     title: "My book",
+        //     author: "Abhijit",
+        //     price: "1000",
+        //     categoryName: "dummy",
+        // },
+    ]);
+
+    useEffect(()=>{
+        (async () => {
+            const res = await axios.get("/api/products/");
```
Comment (sushil-mod) [inline src/context/product-context.js:19]: instead of res you can also directly write {data}

**Review by sushil-mod — APPROVED** [review]
some suggestion , rest code  is good .

**Pull request merged**